### PR TITLE
Add the Footer to the DApp

### PR DESCRIPTION
**Pull request by am9u**
The PR adds a `Footer` component, with styles from the marketing site, and implements it in the DApp.

Issue: #979 

Final links still need to be confirmed by @nguyetv (ie: should the DApp have the same links as the marketing site? Or should they be unique for the dapp per the mock in this issue's ticket?)

### CODE REVIEW

File: packages/components/src/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  currentYear: string;
+}
+
+export const Footer: React.SFC<FooterProps> = props => {
+  return (
+    <StyledFooter>
+      <StyledFooterInner>
+        <StyledFooterInnerLeft>
+          <StyledFooterLogo>
+            <a href="https://civil.co">
```
Comment (nickreynolds) [inline packages/components/src/Footer/Footer.tsx]: don't we have a folder where icons live?

File: packages/components/src/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                </li>
+              </ul>
+            </div>
+            <div>
+              <ul>
+                <li>
+                  <b>About</b>
+                </li>
+                <li>
+                  <a href="https://blog.joincivil.com/" target="_blank">
```
Comment (nickreynolds) [inline packages/components/src/Footer/Footer.tsx]: should all these links be constants / in an enum?

File: packages/components/src/Footer/Footer.tsx (hunk 1 above)
Comment (am9u) [reply packages/components/src/Footer/Footer.tsx]: Ah yeah, I just updated this PR with the proper icons (and added new ones where applicable).

File: packages/components/src/Footer/Footer.tsx (hunk 2 above)
Comment (am9u) [reply packages/components/src/Footer/Footer.tsx]: Yeah, that's a good point. I just got confirmation that these links are the correct ones, so I'll drop them into a constants file.

**Pull request merged**